index.js

```diff
@@ -28,8 +28,7 @@ module.exports = {
                 peerDependencies: false
             }
         ],
-        indent: ['error', 4],
```

Comment:
we're removing indent checking?

Reply:
it's handled by `prettier` both of them have certain incompatible rules. https://github.com/screwdriver-cd/eslint-config-screwdriver/pull/40/files#diff-168726dbe96b3ce427e7fedce31bb0bcL52

Reply:
It's handled here.

```
        'prettier/prettier': [
            'error',
            {
                singleQuote: true,
                tabWidth: 4,
                printWidth: 120
            }
        ],
```

https://github.com/screwdriver-cd/eslint-config-screwdriver/pull/40/files#diff-168726dbe96b3ce427e7fedce31bb0bcR65-R71